package.json

```
@@ -7,7 +7,10 @@
   "scripts": {
     "lint": "yarn run lint:js && yarn run lint:scss",
     "lint:js": "standardx 'app/assets/javascripts/**/*.js' 'spec/javascripts/**/*.js'",
-    "lint:scss": "stylelint app/assets/stylesheets/"
+    "lint:scss": "stylelint app/assets/stylesheets/",
+    "jasmine:prepare": "RAILS_ENV=test bundle exec rails assets:clobber assets:precompile",
+    "jasmine:ci": "yarn run jasmine:prepare && bundle exec rake webdrivers:chromedriver:update && PATH=~/.webdrivers:$PATH yarn run jasmine-browser-runner runSpecs",
```

Shall we go for: 
```suggestion
    "jasmine:ci": "yarn run jasmine:prepare && yarn run jasmine-browser-runner runSpecs",
```
?

I guess we'll get a fail in CI until we install a chromedriver bin there.